syntax = "proto3";

message TranscriptionRequest {
    repeated float data = 2;
}

message TranscriptionResponse {
    int64 buffer_start = 1;
    int64 buffer_end = 2;
    repeated TranscriptionChunk chunks = 3;
    int32 code = 4;
}

message TranscriptionChunk {
    string text = 1;
    int64 start_time = 2;
    int64 end_time = 3;
}
